src/t_hash.c

```diff
@@ -310,6 +311,34 @@ int hashTypeExists(robj *o, sds field) {
     return hashTypeGetValue(o, field, &vstr, &vlen, &vll, NULL) == C_OK;
 }
 
+int hashTypeHasStringRef(robj *o, sds field) {
```

Comment:
This should return a `bool`.  Otherwise, it needs documentation regarding the returned value.